terraform/testnet/forge/templates/cron.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     {{- include "forge.labels" . | nindent 4 }}
 spec:
   schedule: "0 0 * * *"
+  backoffLimit: 1
```

Comment:
Does this make sure it tries again exactly once?

Reply:
yea. removed this actually. the default is 6. this is the job that cleans up old forge jobs